bublik/urls.py

```diff
@@ -59,6 +60,9 @@
     # V2
     re_path(r'^v2/$', api_v2.render_react, name='ui-v2-index'),
     re_path(r'^v2/(?:.*)/?$', api_v2.render_react, name='ui-v2-routes'),
+    path('api/v2/schema/swagger.yaml', SpectacularAPIView.as_view(), name='schema'),
+    path('api/v2/schema/swagger-ui', SpectacularSwaggerView.as_view(url_name='schema'), name='swagger-ui'),
```

Comment:
1. The required slash is missing at the end of the path.
2. The line is too long (will be fixed as a result of applying the linter).